src/pages/dapps.tsx

```diff
@@ -107,8 +107,10 @@ import dai from "@/public/dapps/stabledai.png"
 import status from "@/public/dapps/status.png"
 import superrare from "@/public/dapps/superrare.png"
 import synthetix from "@/public/dapps/synthetix.png"
-import uniswapec from "@/public/dapps/uni.png"
-import uniswap from "@/public/dapps/uni.png"
+import {
+  default as uniswap,
+  default as uniswapec,
+} from "@/public/dapps/uni.png"
```

Comment:
Why are we doing this?

Reply:
That is indeed an auto-fix from eslint. Else the following error throws:
`Module '"*.png"' has no exported member 'uniswapec'. Did you mean to use 'import uniswapec from "*.png"' instead?`

I'm unfamiliar with how this is possible to get multiple named imports from a single image file anyway 🤔

Reply:
@pettinarip that is indeed an auto-fix from eslint, as it is the same image file imported twice. I guess eslint is unaware that it's an image file, so it consolidated the alias naming.

I'll look at throwing up a quick PR to address this duplication.

Reply:
Aha! didn't notice that, good call, thanks. Ok, so I guess we are ok then here 👍🏼